src/w3c/templates/sotd.js

```diff
@@ -338,9 +362,31 @@ function linkToWorkingGroup(conf) {
   if (!conf.wg) {
     return;
   }
+  let proposedChanges = "";
```

Comment:
I think `null` should be ok here... hyperHTML should then remove it.  

```suggestion
  let proposedChanges = null;
```